osu.Game/Screens/Play/Player.cs

```diff
@@ -216,6 +216,8 @@ private void load(AudioManager audio, APIAccess api, OsuConfigManager config)
                 }
             };
 
+            hudOverlay.HoldToQuit.Button.ExitAction = Exit;
```

Comment:
Would rather that the button wasn't exposed. Make an `Action` property in `HoldToQuit` which proxies the passed value into the button. Make the button itself private.